src/sudoers/mod.rs

```diff
@@ -40,7 +40,7 @@ pub struct Error {
 ///
 /// I.e. the Setting modifications in the second part of the tuple only apply for
 /// items explicitly matched by the first part of the tuple.
-type Customiser<T> = (SpecList<T>, Vec<defaults::SettingsModifier>);
+type Customiser<Scope = ConfigScope> = (Scope, Vec<defaults::SettingsModifier>);
```

Comment:
```suggestion
type Customiser<Scope> = (Scope, Vec<defaults::SettingsModifier>);
```

and use `Customiser<ConfigScope>` at the only place where you used plain `Customiser`.